gitlab/metadata.csv

```diff
@@ -25,7 +25,7 @@ gitlab.go_memstats_mspan_sys_bytes,gauge,,byte,,Number of bytes used for mspan s
 gitlab.go_memstats_next_gc_bytes,gauge,,byte,,Number of heap bytes when next garbage collection will take place,0,gitlab,heap bytes next gc
 gitlab.go_memstats_other_sys_bytes,gauge,,byte,,Number of bytes used for other system allocations,0,gitlab,other bytes used
 gitlab.go_memstats_stack_inuse_bytes,gauge,,byte,,Number of bytes in use by the stack allocator,0,gitlab,bytes stack allocator
-gitlab.go_memstats_stack_inuse_bytes,gauge,,byte,,Number of bytes obtained from system for stack allocator,0,gitlab,bytes stack allocator from sys
+gitlab.go_memstats_stack_sys_bytes,gauge,,byte,,Number of bytes obtained from system for stack allocator,0,gitlab,bytes stack allocator from sys
```

Comment:
what happened here?

Reply:
this one is a duplicate of the one directly above, changed named based on the same naming convention of the other ones